### ISSUE TITLE
Subtitle rejected by browsers on ssl conection

### ISSUE THREAD

**Issue by scara78** [Davenchy/live-torrent#46]
Subtitles are still loaded as insecure connection under ssl. Must allow insecure objects in browser settings.
Thank you for such a wonderful project

**Comment by Davenchy** [comment]
OK bro, I am going to support HTTPS for frontend.
But I need to know, are you talking about heroku hosting or local hosting?
And thank you for using Live Torrent.

**Comment by Davenchy** [comment]
I think i got it, I am using Brave browser and the browser blocks the caption file when ever Live Torrent tries to download it and shows "Insecure content blocked" icon in the url bar.
Is this what you are talking about because all what I can do is to create a proxy for OpenSubtitles.org like what I did with YTS api here ddf4a39e3e8cc2eb4f5beb464fe33a3ac15c210f.

**Comment by scara78** [comment]
Yes, this is what im talking about, im using heroku, even installed on localhost subtitles are not loaded.
An api to get movie stream by tmdbid will be nice,if you know what i mean 
someting like http://live-torrent.herokuapp.com/stream/tt73444

Have a nice day!

**Comment by Davenchy** [comment]
OK, thanks for the feedback right now I created opensubtitles.org captions proxy and still working on tests before publishing the backend package and I will try to provide the api like in your example then I will start working on the frontend itself.
Happy Day!

**Comment by dosboxonline** [comment]
Add this `<meta` http-equiv="Content-Security-Policy" `content="upgrade-insecure-requests">` to index.html file.
